packages/cli/src/commands/init/__tests__/editTemplate.test.ts

```diff
@@ -124,3 +124,26 @@ test('should edit template with custom title', () => {
     snapshotDiff(oldJavaFile, newJavaFile, {contextLines: 1}),
   ).toMatchSnapshot();
 });
+
+test('should edit package.json template', () => {
+  jest.spyOn(process, 'cwd').mockImplementation(() => testPath);
```

Comment:
We should clean up after this. Feel free to wrap this test in a describe block and use beforeEach and afterEach to setup and teardown the mock